themes/socialbase/templates/system/page-title.html.twig

```diff
@@ -35,5 +35,5 @@
 
 
 {% if title %}
-  <h1{{ title_attributes.addClass(classes, extra_classes) }}>{{ title }}</h1>
+  <h1{{ title_attributes.addClass(classes, extra_classes) }}><span>{{ title }}</span></h1>
```

Comment:
I'm not sure a span in a title really adds anything here? We should probably just remove it.

Reply:
Fixed